draft/integrating-events-operator/integrating-events-operator.md

```diff
@@ -0,0 +1,347 @@
+---
+permalink: /guides/integrating-events-operator/
+layout: guide-markdown
+title: Integrating the events operator
+duration: 30 minutes
+releasedate: 2020-04-26
+description: Explore how to manage the flow of GitHub events to multiple pipelines
+tags: ['Pipelines', 'Stack']
+guide-category: pipelines
+---
+
+<!-- Note:
+> This repository contains the guide documentation source. To view
+> the guide in published form, view it on the [website](https://kabanero.io/guides/{projectid}.html).
+-->
+
+<!--
+//
+//	Copyright 2020 IBM Corporation and others.
+//
+//	Licensed under the Apache License, Version 2.0 (the "License");
+//	you may not use this file except in compliance with the License.
+//	You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+//	Unless required by applicable law or agreed to in writing, software
+//	distributed under the License is distributed on an "AS IS" BASIS,
+//	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//	See the License for the specific language governing permissions and
+//	limitations under the License.
+//
+-->
+
+## What you'll learn
+
+In this guide, you will learn how to integrate the events operator to manage the flow of GitHub organization events through multiple pipelines. By setting up the webhook mediator, you will manage event flow based on the semantic versioning of your application stack projects.
+
+## Prerequisites
+
+- Install [OpenShift  4.2 or later](https://www.openshift.com/products/container-platform)
+- Install [Kabanero  Foundation](../../docs/ref/general/installation/installing-kabanero-foundation.html)
+
+
+## Introduction
+
+The events operator allows users to define an event mediation flow for Kubernetes deployments. By using custom resource definitions (CRD),
+users can quickly construct mediation logic to receive, transform, and route JSON data structures. The transformation logic
+is based on the Common Expression Language (CEL).
+
+To integrate the events operator with Kabanero, you must use a function of the mediator framework, known as the webhook mediator, to process webhook events. The webhook mediator is used with a GitHub organization webhook to drive the pipelines.
+
```

Comment:
"You may use an event mediator to process webhook events. A mediator may be configured to drive versioned pipelines based on  Github organization webhooks events for typed repositories, such as Appsody repositories"

Reply:
Hi @mchenggit - I think we need to simplify the language for the end user. Your suggestion has helped me understand a bit more, so I'm hoping that the following might be a good replacement for the original paragraph:

_To integrate the events operator with Kabanero, you use an event mediator to process events from a GitHub organization webhook. The event mediator is configured to filter and direct the flow of these events from different application stack repositories to the appropriate semantically-versioned pipeline. This process enables you to configure which events trigger which pipelines for making updates to different Kubernetes deployments._

What do you think?

Reply:
That is good. However, it's not the only integration. Though it is the only one we will demonstrate at the moment.

Reply:
OK I'll add "you might use..." which implies an alternative, even if this does leave them wondering what that is ;-)